spatialmedia/gui.py

```diff
@@ -0,0 +1,118 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Spherical Metadata Python Tool
+
+Tool for examining and injecting spherical metadata into MP4 files.
+Naming conventions are styled to match TKinter naming conventions.
```

Comment:
I think you can drop this comment as your naming below is both PEP-8 and Google style complaint. You renamed createWidgets -> create_widgets, making it all happy.